service/src/main/java/bio/terra/profile/service/profile/model/ProfileDescription.java

```diff
@@ -2,31 +2,36 @@
 
 import bio.terra.policy.model.TpsPolicyInputs;
 import bio.terra.profile.model.CreateProfileRequest;
+import bio.terra.profile.model.Organization;
 import bio.terra.profile.model.ProfileModel;
 import bio.terra.profile.service.policy.TpsConversionUtils;
 import java.util.Objects;
 import java.util.Optional;
 
 public record ProfileDescription(
-    BillingProfile billingProfile, Optional<TpsPolicyInputs> policies) {
+    BillingProfile billingProfile,
+    Optional<TpsPolicyInputs> policies,
+    Optional<Organization> organization) {
   public ProfileDescription {
     Objects.requireNonNull(billingProfile);
     Objects.requireNonNull(policies);
   }
 
   public ProfileDescription(BillingProfile billingProfile) {
-    this(billingProfile, Optional.empty());
+    this(billingProfile, Optional.empty(), Optional.empty());
   }
 
   public static ProfileDescription fromApiCreateProfileRequest(CreateProfileRequest request) {
     return new ProfileDescription(
         BillingProfile.fromApiCreateProfileRequest(request),
-        Optional.ofNullable(TpsConversionUtils.tpsFromBpmApiPolicyInputs(request.getPolicies())));
+        Optional.ofNullable(TpsConversionUtils.tpsFromBpmApiPolicyInputs(request.getPolicies())),
+        Optional.empty());
```

Comment:
Don't we want to try to figure out of if the subscription is in the enterprise list here?

Reply:
It's a bit awkward to do here since we don't have access to the `EnterpriseConfiguration` needed to determine if the `CreateProfileRequest` will be an enterprise organization. I'm looking into refactoring `ProfileDescription` to see if I can make it a little easier

Reply:
I haven't been able to refactor this into something better. I'm inclined to leave it as is since the `CreateProfileRequest` doesn't contain any organization information as it is. The `CreateProfileFlight` has been changed to return the organization information after creating the profile

Reply:
I am not sure if a `record` with multiple `Optional` fields is the best way to represent the `ProfileDescription` or if there's a simpler way to handle the `EnterpriseConfiguration`, but I haven't been able to come up with better alternatives. Open to suggestions on this